### DISCUSSION TITLE
Увеличить время смерти до 24х часов

### DISCUSSION THREAD

**Discussion by KillerBunny031** [octothorp-team/public-dobrograd#670] [Предложения]
> Вводная часть 

(Говорю конкретно о Новом Доброграде, ибо на Центральном я играю редко и не знаю ситуацию, которая там происходит) 
В Доброграде процветают бандитские группировки. Сейчас, если ты выедешь за город, то тебя сразу же подобьют, ограбят и в придачу обоссут. Да что уж там! Даже в городе ты не можешь чувствовать себя в безопасности. И я сейчас говорю не про Рузвельт! То же здание на Бравери 1. Там часто происходят ограбления и убийства.
И что же происходит, когда на вызов о преступлении приезжает полиция? Правильно, начинается перестрелка. Не важно сколько полицейских и что за вооружение у преступника. Стрельба все равно будет, потому что ни одна из сторон не боится умереть. 

> Так что же я предлагаю?

Я предлагаю провести эксперемент. Увеличить время смерти до 24х реальных часов, вместо 30 минут

> К чему это должно все привести?

Ну, должна повысится ролевая составляющая, потому что люди реально начнут бояться умирать. Преступники начнут сдаваться, вместо того, чтобы гибнуть в неравном бою. Офицеры же начнут более тчательно продумывать тактики устранения преступников, вместо того, чтобы лезть напролом.

> Что, если эксперемент провалится?

В таком случае время смерти можно откатить обратно на 30 минут.

**Comment by Black-White228337** [comment]
По сути ты предлагаешь давать бан на день за свою смерть, иногда если что смерти происходят не в перестрелках, вот например я играю роль маньяка, убью я человека и из-за этого ему ждать день? Человек может умереть из-за бага, или банально разбиться на машине из-за пинга. Это исправит только онлайн на сервере, и только в худшую сторону так как большинство людей которые как либо делают РПшки, теряют деньги, оружия, еще должны сидеть 24 часа на сервере просто выйдут нахрен с сервера, и что по твоему кому-то это в кайф вообще будет? А если война банд или каких нибудь других криминальных группировок? И все ради того чтобы какой нибудь чел который хочет постреляться просто выбросил оружие вместо того чтобы умирать

**Comment by Dawaren** [comment]
Против. Ты предлагаешь людям действительно оставлять компьютер включенным на 24 часа? А если человека убили без причины и админов нет на сервере?

**Comment by Kyper1** [comment]
Против, не подумал о RDMщиках, обходчиках; игроках, которых убили случайно.

**Comment by LENFYS** [comment]
Максимум смерть персонажа, не более.

**Comment by travka4000** [comment]
Конечно же против это глупо
